Objects/memoryobject.c

```diff
@@ -2748,6 +2748,50 @@ static PySequenceMethods memory_as_sequence = {
 };
 
 
+/****************************************************************************/
+/*                              Counting                                    */
+/****************************************************************************/
+
+/*[clinic input]
+memoryview.count
```

Comment:
```suggestion
memoryview.count -> Py_ssize_t
```

And return `make clinic`

Reply:
Is it really needed? list.count() does not seem to use this annotation.

Reply:
For now, I think I'll avoid using the annotation. With the annotation, while I can return -1, there will be a bit more work at runtime since there will be some `PyErr_Occurred()` checks and others. Instead, I can directly construct the integer and return it.

Reply:
IMO, we can reduce some boilerplate if we use return type through AC, but it's up to you.

Reply:
But yes, we should raise some error if we make same effect of return NULL.